.gitignore

```diff
@@ -7,8 +7,8 @@ src/bitcoind
 src/bitcoin-cli
 src/bitcoin-tx
 src/bitcoin-wallet
+src/test/fuzz
```

Comment:
This is a folder, as opposed to the ` src/test/test_bitcoin_fuzzy` binary.

I am not sure if the whole folder should be excluded. Also I am not sure if it makes sense to enumerate all binaries one-by-one.

No strong opinion, though. I don't really care about gitignore.

Reply:
> No strong opinion, though. I don't really care about gitignore.

Out-of-tree builds ftw.

But I think as long as we're enumerating binaries here, we should add them all. Ignoring the entire directory is a bad idea because it will also ignore new source files.

Reply:
Suggest adding `!src/test/fuzz/*.*` below this line to re-watch anything with an extension. This on its own would start ignoring even .cpp files in that folder.

Reply:
Agreed! I was hesitant between just removing `src/test/test_bitcoin_fuzzy` or adding the unignore.

Reply:
I'll pull your commit into this PR.